apps/component-date/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  // this will mean you will need to go to http://localhost:8080/component-date
```

Comment:
Only if your app is running at `http://localhost:8080`. IMO, you should either be clear about the address or not mentions this at all.